src/libYARP_OS/include/yarp/os/Time.h

```diff
@@ -69,13 +78,23 @@ class YARP_OS_API yarp::os::Time {
      *
      * \see yarp::os::NetworkClock
      *
+     * return true on success, false on failure. Possible causes of
+     * failure are invalid port name or address conflict.
+     *
+     * Throws assert in case of failure
```

Comment:
+1, but I'll merge anyway because it's time! 😆 
@barbalberto please fix this comment later!